internal/build/templates/ensure-sysext.service

```diff
@@ -9,12 +9,15 @@ ConditionDirectoryNotEmpty=|/run/extensions
 ConditionDirectoryNotEmpty=|/var/lib/extensions
 ConditionDirectoryNotEmpty=|/usr/local/lib/extensions
 ConditionDirectoryNotEmpty=|/usr/lib/extensions
+ConditionPathExists=/usr/bin/elemental3ctl
```

Comment:
This is implicitly making elemental3ctl extension a hard requirement to load any service provided by an extension. I agree with @dirkmueller that `elemental3ctl kmod` should be handled separately in its own unit and do not have any kind dependency with `ensure-sysext` and only depend on `systemd-sysext` service, in fact, the two could run in parallel.

I am fine leaving it like this for now, but IMHO this requires  follow up PR.